package.json

```diff
@@ -33,13 +33,13 @@
     "lerna": "^3.22.0",
     "lint-staged": "^10.4.0",
     "prettier": "^2.1.2",
+    "qawolf": "^1.4.0",
```

Comment:
@ali-fs , is it possible to update `qawolf` to version `1.7.0`?
The existing versions don't support `arm_64` so I find myself having to update manually on my local branch each time. Would be great if we can update it on `master`.

Reply:
Let's keep the previous version, we are removing the qawolf entirely very soon.

Reply:
#2644 will  totally remove `qawolf`